scripts/create-sentry-release

```diff
@@ -29,21 +29,30 @@ if [ -z "${SENTRY_ORG:-}" ] || [ -z "${SENTRY_PROJECT:-}" ]; then
 fi
 
 echo 'Downloading debug info, source bundle, system symbols...'
-gsutil cp \
-  "gs://dicd-team-devinfra-cd--relay/deployment-assets/${REVISION}/${NAME}/release-name" \
-  "gs://dicd-team-devinfra-cd--relay/deployment-assets/${REVISION}/${NAME}/relay-debug.zip" \
-  "gs://dicd-team-devinfra-cd--relay/deployment-assets/${REVISION}/${NAME}/relay.src.zip" \
-  "gs://dicd-team-devinfra-cd--relay/deployment-assets/${REVISION}/${NAME}/libs.tar" \
-  .
-
-echo 'Uploading debug information and source bundle...'
-sentry-cli --version
-sentry-cli upload-dif ./relay-debug.zip ./relay.src.zip
-
-echo 'Uploading system symbols...'
-tar xf libs.tar
-sentry-cli upload-dif lib/x86_64-linux-gnu
-
+for PLATFORM in "linux/amd64" "linux/arm64"; do
+  if [ "$PLATFORM" = "linux/amd64" ]; then
+    LIB_PATH="lib/x86_64-linux-gnu"
+  else
+    LIB_PATH="lib/aarch64-linux-gnu"
+  fi
```

Comment:
Is this necessary? Is uploading `lib/` not good enough?

Reply:
Good point, quick check later seems like lib is enough.